### PR TITLE
Feat: Settings app UI

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- update Settings app UI
- improve DS components
- update legal app routes

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/design-system-core/src/components/Box/index.tsx [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+  onClick?: () => void;
 }
 
 const Box: React.FC<PropsWithChildren<IBoxProps>> = props => {
-  const { customStyle = '', children } = props;
+  const { customStyle = '', onClick, children } = props;
 
-  return <div className={tw(customStyle)}>{children}</div>;
+  // if onClick is defined, use cursor pointer, unless otherwise
+  const cursorStyle = `cursor-${typeof onClick === 'function' ? 'pointer' : 'default'}`;
```
Comment (didd) [inline ui/design-system-core/src/components/Box/index.tsx:13]:
```suggestion
  const cursorStyle = `cursor-${onClick ? 'pointer' : 'default'}`;
```
`onClick` is already determined as a function from the type definition

File: ui/design-system-core/src/components/Box/index.tsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
 import React, { PropsWithChildren } from 'react';
-import { tw } from '@twind/core';
+import { apply, tw } from '@twind/core';
 
 export interface IBoxProps {
   customStyle?: string;
+  onClick?: () => void;
```
Comment (didd) [inline ui/design-system-core/src/components/Box/index.tsx]:
```suggestion
```
You can use the semantically relevant button component in this case with plain prop as `<Button onClick={...} plain> ... </Button>`.  You can [read](https://www.builder.io/blog/buttons) here why we prefer using button over div for handling `onClick` events.

File: ui/design-system-core/src/components/Anchor/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { PropsWithChildren } from 'react';
+import { apply, tw } from '@twind/core';
+
+export interface IAnchorProps {
```
Comment (didd) [inline ui/design-system-core/src/components/Anchor/index.tsx:4]:
```suggestion
export interface IAnchorProps & React.DetailedHTMLProps<React.AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement> {
```

File: ui/design-system-core/src/components/Anchor/index.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  href: string;
```
Comment (didd) [inline ui/design-system-core/src/components/Anchor/index.tsx]:
```suggestion
```

File: ui/design-system-core/src/components/Anchor/index.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  customStyle?: string;
+  target?: '_self' | '_blank' | '_parent' | '_top';
```
Comment (didd) [inline ui/design-system-core/src/components/Anchor/index.tsx]:
```suggestion
```

File: ui/design-system-core/src/components/Anchor/index.tsx (hunk 4 above)
Comment (didd) [reply ui/design-system-core/src/components/Anchor/index.tsx]: https://github.com/AKASHAorg/akasha-framework/pull/1886/files#r1145730772

File: ui/design-system-core/src/components/Anchor/index.tsx (hunk 5 above)
Comment (didd) [reply ui/design-system-core/src/components/Anchor/index.tsx]: https://github.com/AKASHAorg/akasha-framework/pull/1886/files#r1145730772

File: ui/design-system-core/src/components/Box/index.tsx (hunk 2 above)
Comment (josenriagu) [reply ui/design-system-core/src/components/Box/index.tsx]: Alright. Will take a look

**Review by didd — APPROVED** [review]

**Pull request merged**